app/views/layouts/application.html.erb

```diff
@@ -9,7 +9,11 @@
     <body>
       <div id="wrapper"> 
         <%= render 'layouts/header' %> 
+        <% flash.each do | message| %> 
+          <div><%= message %></div>  
+        <% end %> 
         <%= yield %>
+        <%= debug(params) if Rails.env.development? %> 
```

Comment:
if you gain value from seeing the output of this, that's great. if not - I'd remove it.